### PR TITLE
I want to see a confirm dialog when I try to delete any item

### PR DESCRIPTION
**Pull request by elbosio**

### CODE REVIEW

File: grails-app/domain/io/tarmac/pregnancyControl/SingleMeasurementDef.groovy [hunk 1]
```diff
@@ -12,7 +12,7 @@ class SingleMeasurementDef extends AbstractMeasurementDef{
     GoalType goalType
 
     BigDecimal minValue = 0
-    BigDecimal maxValue = BigDecimal.valueOf(Double.MAX_VALUE)
+    BigDecimal maxValue
```
Comment (ppazos) [inline grails-app/domain/io/tarmac/pregnancyControl/SingleMeasurementDef.groovy]: @elbosio does this validates if maxValue is null? maxValue is not nullable.

File: grails-app/services/simplecare/ConditionService.groovy [hunk 2]
```diff
@@ -11,6 +11,12 @@ class ConditionService {
     def setConditionToPatient(Condition condition, Patient patient) {
         if (condition.id == null) {
             patient.addToConditions(condition)
+            condition.conditionDef?.measurementDefs.each {
+                if (!(it in patient.measuresDefinitions)) {
+                    patient.addToMeasuresDefinitions(it)
```
Comment (ppazos) [inline grails-app/services/simplecare/ConditionService.groovy]: @elbosio does the "in" compares by object or by id?

File: grails-app/views/measureDef/_form.gsp [hunk 3]
```diff
@@ -21,19 +21,7 @@
         <div class="form-group">
             <label class="control-label col-sm-2 control-label"> ${message(code: 'measureDef.category.label')}</label>
             <div class="col-sm-8">
-                <g:select name="categoryId" class="form-control" placeholder="${message(code: 'measureDef.category.label')}" from="${simplecare.MeasurementDefCategory.list([sort: 'name', order: 'asc'])}" optionKey="id" optionValue="name" value="${measureDefInstance?.category?.id}"/>
-            </div>
-        </div>
-        <div class="form-group">
-            <label class="control-label col-sm-2 control-label"> ${message(code: 'measureDef.minValue.label')}</label>
-            <div class="col-sm-8">
-                <g:textField name="minValue" required="required" class="form-control" placeholder="${message(code: 'measureDef.minValue.label')}" value="${measureDefInstance.minValue}"></g:textField>
```
Comment (ppazos) [inline grails-app/views/measureDef/_form.gsp]: @elbosio shouldn't we show the min / max values?

**Comment by elbosio** [comment]
@ppazos Yes it is. We only check minValue =< maxValue. We used to have Long.MAX_VALUE as a default value but looks awful at UI.

File: grails-app/services/simplecare/ConditionService.groovy (hunk 2 above)
Comment (elbosio) [inline grails-app/services/simplecare/ConditionService.groovy]: @ppazos Last time we check... Nope. But for some reason works fine. I'll change it just to be sure.

**Comment by ppazos** [comment]
@elbosio +1

**Comment by machador** [comment]
+1 :shipit:

**Pull request merged**